### ISSUE TITLE
What the "CK" means in eval.py?

### ISSUE THREAD

**Issue by Yustarzzz** [daveredrum/Pointnet2.ScanNet#10]
Hi, @daveredrum
I'm happy to meet your wonderful code!

I have a question about 'CK''s meaning in eval.py. 
I think N is num of points and C is num of class.
However, I have no idea about CK. 
Can you help me?
thanks. 

**In eval.py**
def forward(args, model, coords, feats):

    pred = []

    coord_chunk, feat_chunk = torch.split(coords.squeeze(0), args.batch_size, 0), torch.split(feats.squeeze(0), args.batch_size, 0)

    assert len(coord_chunk) == len(feat_chunk)

    for coord, feat in zip(coord_chunk, feat_chunk):

        output = model(torch.cat([coord, feat], dim=2))

        pred.append(output)

    pred = torch.cat(pred, dim=0).unsqueeze(0) # (1, CK, N, C)

**Comment by daveredrum** [comment]
Hi @Yustarzzz, "CK" here is short for "Chunk".
